src/elastio-snap.c

```diff
@@ -1836,23 +1840,26 @@ int file_write_block(struct snap_device *dev, void *block, size_t offset, size_t
 	bdev = dev->sd_base_dev;
 	sectors_processed = 0;
 
-	WARN_ON(!IS_ALIGNED(offset, PAGE_SIZE) || len > SECTORS_PER_BLOCK);
+	WARN_ON(len > SECTORS_PER_BLOCK);
```

Comment:
Offset may be unaligned on architectures where PAGE_SIZE is more than 4k (f.e., on CentOS 8 + aarch64, where it's 64k)